app-admin/mkosi/mkosi-11.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit distutils-r1 optfeature
+
+DESCRIPTION="Build Bespoke OS Images"
+HOMEPAGE="https://github.com/systemd/mkosi"
+SRC_URI="https://github.com/systemd/mkosi/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+debian"
+RDEPEND="
+	sys-apps/portage
+	sys-apps/systemd
+	dev-vcs/git
+	debian? ( dev-util/debootstrap )
```

Comment:
Remove the debian use flag and this dependency declaration. Using optfeature is enough.